### DISCUSSION TITLE
Unable to setup database

### DISCUSSION THREAD

**Discussion by Castafiore** [directus/directus#4311] [Archive]
Hi, I've installed Directus on my vm, and I have setup several databases several months ago. Recently I want to setup a new database which is hosted on same server as previous already setup databases, but I found I cannot see the interfaces to setup a database from Directus webapp anymore. It seems like the directus has been upgraded and I cannot find the same documentations anymore. Could you tell me how to setup a database from Directus webapp or via CLI?

**Comment by rijkvanzanten** [comment]
Heya! 

I think this is what you're looking for:

https://docs.directus.io/getting-started/quickstart/

**Reply by Castafiore** [reply]
Hi Rijk,

I tried several times but stuck on Install Directus. Is there a way to do it on web app interface just like before?

**Reply by rijkvanzanten** [reply]
No, the CLI is the way to install/migrate/initialize the database: https://docs.directus.io/reference/command-line-interface/

**Reply by Castafiore** [reply]
But the stale way worked for me, the CLI way doesn't work for me. It is always Installing Directus.

**Reply by rijkvanzanten** [reply]
What's the issue?
